cmake/extensions.cmake

```
@@ -2760,6 +2780,17 @@ function(dt_reg_size var)
   set(${var} ${${var}} PARENT_SCOPE)
 endfunction()
 
+# Internal helper for dt_reg_addr/dt_reg_size; not meant to be used directly
+function(dt_reg_index_private var path name)
+  dt_prop(reg_names PATH "${path}" PROPERTY "reg-names")
+  if(NOT DEFINED reg_names)
+    set(index "-1")
+  else()
+    list(FIND reg_names "${name}" index)
+  endif()
+  set("${var}" "${index}" PARENT_SCOPE)
```

Why the quotes on the return var ?

If there is any risk that `${var}` can be undefined or empty string, then we should not try to do a `set()` on it, but based on how this is used, that is not really a risk.
```suggestion
  set(${var} "${index}" PARENT_SCOPE)
```

---

@mbolivar-nordic why was this marked resolved without any comments or fixes ?